### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by DmitriyMV**
Bump stuff:
- run rekres
- bump bldr
- binutils-gdb to 2.43
- linux to 6.6.44

### CODE REVIEW

**Review by frezbo — APPROVED** [review]

File: binutils/pkg.yaml [hunk 1]
```diff
@@ -35,6 +35,7 @@ steps:
     build:
       - |
         cd build
+        export CFLAGS_FOR_TARGET="-Wno-error=implicit-function-declaration"
```
Comment (DmitriyMV) [inline binutils/pkg.yaml:38]: _Backward compatibility they say_

**Comment by smira** [comment]
I saw too many build failures with gcc 14.2 - let's not get into that for 1.8 please.

Keep gcc as it was.

**Comment by DmitriyMV** [comment]
> Keep gcc as it was.

Done!

**Pull request merged**